client/src/app/tabs/cloud-dmn/__tests__/DmnEditorSpec.js

```diff
@@ -1359,115 +1360,129 @@ describe('<DmnEditor>', function() {
 
       instance.handleChanged();
 
-      wrapper.update();
-
-      const toggle = wrapper.find('#button-toggle-overview');
-
       // when
-      toggle.simulate('click');
+      const toggle = document.querySelector('#button-toggle-overview');
+      fireEvent.click(toggle);
 
       // then
-      expect(layout.dmnOverview.open).to.be.false;
+      await waitFor(() => {
+        expect(onLayoutChanged).to.have.been.calledOnce;
+      });
+
+      const callArg = onLayoutChanged.getCall(0).args[0];
+      expect(callArg).to.deep.include({
+        dmnOverview: {
+          open: false
+        }
+      });
     });
 
 
     it('should open properties panel (no layout)', async function() {
 
       // given
-      let layout = {};
+      const layout = {};
 
-      function onLayoutChanged(newLayout) {
-        layout = newLayout;
-      }
+      const onLayoutChanged = sinon.spy();
 
-      const {
-        wrapper
-      } = await renderEditor(diagramXML, {
+      await renderEditor(diagramXML, {
```

Comment:
What happened to this pattern:

```suggestion
const {
  find...
} = renderEditor(..)

// later use the local testing util to find the element.
find(...)
```

Reply:
This has never been implemented? I can't find any usage of this in BpmnEditorSpec. Not saying that we couldn't make any use of it.